main.go

```diff
@@ -18,10 +19,10 @@ type config struct {
 func parseConfig() (config, error) {
 	cfg := config{}
 	pflag.StringVarP(&cfg.Addr, "addr", "a", ":9210", "Address to listen on.")
```

Comment:
Just for completeness sake, I'd also add an environment variable for the address. Maybe `NETATMO_EXPORTER_ADDR`?